deploying/deploying-to-heroku.md

```diff
@@ -75,16 +75,16 @@ At this point, you should have the following directory structure:
 ```
 ./eventonica-app
 ./eventonica-app/client/* # The code for your React App
-./eventonica-app/server/* # Your express API (app.js etc.)
-./eventonica-app/package.json # Toplevel package.json used by Heroku to run your app
-./eventonica-app/package-lock.json # Toplevel package-lock.json used by Heroku to run your app
+./eventonica-app/server/* # Your express API (app.js)
+./eventonica-app/package.json # Top level package.json used by Heroku to run your app
+./eventonica-app/package-lock.json # Top level package-lock.json used by Heroku to run your app
 ```
 
-3. Test out your new server locally:
+3. Test out your new server locally.
 
 ```
-# Make sure you use the filename you used when you created your Express API
-node server/app.js
+# Make sure you use the <filename> you used when you created your Express API
+node server/<filename.js
```

Comment:
```suggestion
node server/filename.js
```

Reply:
I was intending <filename> in all instances for consistency. As long as it's consistent then good.